lib/shoehorn/application/event_supervisor.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Shoehorn.Application.EventSupervisor do
```

Comment:
Regarding naming, this is an OTP Application `ReportHandler`, right? That sounds better to me than something called an Event or a Supervisor, since I don't think that it's either an event or a Supervisor from what I've read so far.

Reply:
Its a "gen_event" handler that is used to supervise/watch the apps that are running.  The confusing part is that this gen_event handler gets installed into the error_logger